CMakeLists.txt

```diff
@@ -93,6 +93,10 @@ spirv_cross_add_library(spirv-cross-cpp spirv_cross_cpp STATIC
     ${CMAKE_CURRENT_SOURCE_DIR}/spirv_cpp.hpp
     ${CMAKE_CURRENT_SOURCE_DIR}/spirv_cpp.cpp)
 
+spirv_cross_add_library(spirv-cross-reflect spirv_cross_reflect STATIC
```

Comment:
I'm missing an update to the MSVC project (normally it's built through cmake, just add the few lines required by hand.

Reply:
fixed

Reply:
fixed